tests/MongoDB.Bson.Tests/Jira/CSharp275Tests.cs

```diff
@@ -83,35 +83,36 @@ public Test(string json, string iso)
             new Test("Mon, 10 Oct 2011 11:22:33 MDT", "2011-10-10T11:22:33-06:00"),
             new Test("Mon, 10 Oct 2011 11:22:33 PST", "2011-10-10T11:22:33-08:00"),
             new Test("Mon, 10 Oct 2011 11:22:33 PDT", "2011-10-10T11:22:33-07:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 A", "2011-10-10T11:22:33-01:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 B", "2011-10-10T11:22:33-02:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 C", "2011-10-10T11:22:33-03:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 D", "2011-10-10T11:22:33-04:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 E", "2011-10-10T11:22:33-05:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 F", "2011-10-10T11:22:33-06:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 G", "2011-10-10T11:22:33-07:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 H", "2011-10-10T11:22:33-08:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 I", "2011-10-10T11:22:33-09:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 K", "2011-10-10T11:22:33-10:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 L", "2011-10-10T11:22:33-11:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 M", "2011-10-10T11:22:33-12:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 N", "2011-10-10T11:22:33+01:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 O", "2011-10-10T11:22:33+02:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 P", "2011-10-10T11:22:33+03:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 Q", "2011-10-10T11:22:33+04:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 R", "2011-10-10T11:22:33+05:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 S", "2011-10-10T11:22:33+06:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 T", "2011-10-10T11:22:33+07:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 U", "2011-10-10T11:22:33+08:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 V", "2011-10-10T11:22:33+09:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 W", "2011-10-10T11:22:33+10:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 X", "2011-10-10T11:22:33+11:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 Y", "2011-10-10T11:22:33+12:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 Z", "2011-10-10T11:22:33-00:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 +0000", "2011-10-10T11:22:33+00:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 -0000", "2011-10-10T11:22:33-00:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 +0100", "2011-10-10T11:22:33+01:00"),
-            new Test("Mon, 10 Oct 2011 11:22:33 -0100", "2011-10-10T11:22:33-01:00")
+            // TODO: Investigate Military Time Failures
```

Comment:
Note CSHARP-3268 as the ticket to investigate these failures.

Reply:
Added code comment with link.

Reply:
Perfect.